api/src/main/java/org/apache/cloudstack/api/response/NetworkOfferingResponse.java

```diff
@@ -124,6 +123,22 @@
     @Param(description = "true if network offering supports public access for guest networks", since = "4.10.0")
     private Boolean supportsPublicAccess;
 
+    @SerializedName(ApiConstants.DOMAIN_ID)
+    @Param(description = "the domain ID(s) this disk offering belongs to. Ignore this information as it is not currently applicable.")
```

Comment:
said to be multiple but the getter does not seem to facilitate this

Reply:
Multiple domains are supported with a comma-separated list of domain IDs. This is done to maintain backward compatibility and consistency. ServiceOfferingResponse and DiskOfferingResponse already had a domainId parameter and if an array of IDs would have been used instead of comma-separated string it might have broken automation scripts for some.